games-action/openclonk-7.0.recipe

```diff
@@ -0,0 +1,127 @@
+SUMMARY="A free multiplayer action game where you control clonks"
+DESCRIPTION="OpenClonk is a free multiplayer action game in which you control \
+clonks, small, but witty and nimble humanoid beings. The game is mainly about \
+mining, settling and fast-paced melees."
+HOMEPAGE="http://www.openclonk.org/"
+COPYRIGHT="2009-2016 The OpenClonk Team and contributors
+	2006-2007 Alexander Post
+	2004-2013  Armin Burgmeier
+	2005-2006, 2008-2010  Asmageddon
+	2010, 2012  Benjamin Herr
+	2006, 2012  Bernhard Bonigl
+	2009-2010  Carl-Philip Hänsch
+	2001  Carlo Teubner
+	2009, 2011-2013  David Dormagen
+	2011-2012  Felix Wagner
+	2006  Florian Groß
+	2004-2012  Günther Brammer
+	2006-2008  Julian Raschke
+	2010-2012  Julius Michaelis
+	2010-2011  Maikel de Vries
+	2010  Manuel Riecke
+	2009  Mark Haßelbusch
+	2009-2012  Martin Plicht
+	1998-2009  Matthes Bender
+	2001  Michael Käser
+	2009  mizipzor
+	2009-2013  Nicolas Hake
+	2013  Oliver Schneider
+	2010  Peewee
+	2001-2011  Peter Wortmann
+	2012  Philipp Kern
+	2009-2010  Richard Gerum
+	2001-2013  Sven Eberhardt
+	2004-2005, 2009-2013  Tobias Zwick"
+LICENSE="ISC"
+SOURCE_URI="http://www.openclonk.org/builds/release/$portVersion/openclonk-$portVersion-src.tar.bz2"
+CHECKSUM_SHA256="bc1a231d72774a7aa8819e54e1f79be27a21b579fb057609398f2aa5700b0732"
+SOURCE_DIR="openclonk-$portVersion-src"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	openclonk$secondaryArchSuffix = $portVersion
+	app:OpenClonk$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libjpeg_turbo$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libxrandr$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLEW$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libault$secondaryArchSuffix
+	lib:libat_1.0k$secondaryArchSuffix
+	lib:libcairo_gobject$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libgdk_pixbuf_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgthread_2.0$secondaryArchSuffix
+	lib:libixml$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libpango_1.0$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libthreadutil$secondaryArchSuffix
+	lib:libtinyxml$secondaryArchSuffix
+	lib:libupnp$secondaryArchSuffix
+	lib:libvorbisfile$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES"
+	haiku_devel$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
```

Comment:
```diff
-	devel:libpng$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
```
(It is safer to use  png 1.6 devel, in case png 1.5 devel is also available)

Hmm, no need to add libpng16_devel here, as we already have it, in line 103.
You should just remove `devel:libpng$secondaryArchSuffix`.

BTW, if you should also sort all the `devel:lib*` lines in `BUILD_REQUIRES`.